server/src/test/java/org/elasticsearch/search/SearchServiceSingleNodeTests.java

```diff
@@ -0,0 +1,3126 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+package org.elasticsearch.search;
+
+import org.apache.lucene.index.DirectoryReader;
```

Comment:
The diff is hard to follow, but I just renamed the existing SearchServiceTests to SearchServiceSingleNodeTests, and added a new test class with unit tests only, which I called SearchServiceTests

Reply:
Thanks for the review aid here